python/bindings.cpp

```diff
@@ -336,7 +346,11 @@ PYBIND11_MODULE(libsonata, m) {
 
         .def("__ne__", &bbp::sonata::operator!=, "Compare selection contents are not equal");
 
-    bindPopulationClass<NodePopulation>(m, "NodePopulation", "Collection of nodes with attributes");
+    auto node_population = bindPopulationClass<NodePopulation>(
+        m, "NodePopulation", "Collection of nodes with attributes");
+    defMatchValues<int>(node_population);
+    defMatchValues<double>(node_population);
```

Comment:
Now that @wizmer commented on it, I can't unsee this. Can we remove the match on floating point numbers? I agree that minimizes the risk of things going the wrong way…